I saw Scratch doesn't work for the build for this PR - https://staging-editor-static.raspberrypi.org/branches/1438_merge/web-component.html


> react-dom.production.min.js:2 Error: Error in Scratch GUI [location=https://staging-editor-static.raspberrypi.org/branches/1438_merge/scratch.html?project_id=cool-scratch.json&api_url=https%3A%2F%2Fstaging-editor-static.raspberrypi.org%2Fbranches%2F1438_merge&scratchMetadata=1&parent_origin=https%3A%2F%2Fstaging-editor-static.raspberrypi.org]: EvalError: Evaluating a string as JavaScript violates the following Content Security Policy directive because 'unsafe-eval' is not an allowed source of script: script-src 'self' 'unsafe-inline'".

--- a/src/scratch.html
+++ b/src/scratch.html
@@ -9,7 +9,7 @@
         default-src 'self';
         base-uri 'none';
         object-src 'none';
-        script-src 'self' 'unsafe-inline' <%= isDev ? "'unsafe-eval'" : "" %>;
+        script-src 'self' 'unsafe-inline' 'unsafe-eval';
         style-src 'self' 'unsafe-inline';
         worker-src 'self' blob:;
         child-src 'self' blob:;

